hack/minikube-start.sh

```diff
@@ -11,8 +11,8 @@
 # */
 
 # Get CPU and Memory from environment variables or default
-CPU_COUNT="${VSECM_MINIKUBE_CPU_COUNT:-8}"
-MEMORY="${VSECM_MINIKUBE_MEMORY:-11264m}"
+CPU_COUNT="${VSECM_MINIKUBE_CPU_COUNT:-2}"
+MEMORY="${VSECM_MINIKUBE_MEMORY:-4096m}"
```

Comment:
The defaults were too high (even for our build server), reduced them to a more sensible level.